examples/minimal/astro.config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	renderers: [],
```

Comment:
For another PR, but I really want to make this the default soon, so this file wouldn't be necessary either.

Reply:
I may be wrong, but an empty array might trigger their installation anyway. Therefore, with your approval, I’d like to remove it as well, and move my +1 over to your suggestion that "no renderers" be the default.

Reply:
I do think it's about time to do that, but not in the scope of this PR. 

Let's leave this file for now. A todo item for me is to add some warnings about the upcoming behavior change to the next version so we can make the switch in the version after that.

Reply:
If an empty array doesn’t currently limit the renderers, should I still keep this line? Not sure if I’m to:

- **A**. Keep the line. It has an effect.
- **B**. Remove the line. It has no effect.
- **C**. Remove the file, because the line has no effect.

If the line has an effect, then I would vote **A**, but if it has no effect then I’m on the fence between **B** and **C**.